src/main/java/it/smartcommunitylab/aac/spid/provider/SpidIdentityProviderConfigMap.java

```diff
@@ -0,0 +1,295 @@
+import it.smartcommunitylab.aac.base.model.AbstractConfigMap;
+import it.smartcommunitylab.aac.spid.model.SpidAttribute;
+import it.smartcommunitylab.aac.spid.model.SpidAuthnContext;
+import it.smartcommunitylab.aac.spid.model.SpidUserAttribute;
+import java.io.Serializable;
+import java.util.Map;
+import java.util.Set;
+import javax.validation.Valid;
+
+@Valid
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SpidIdentityProviderConfigMap extends AbstractConfigMap implements Serializable {
+
+    private static final long serialVersionUID = SystemKeys.AAC_SPID_SERIAL_VERSION;
+    public static final String RESOURCE_TYPE =
+        SystemKeys.RESOURCE_CONFIG +
+        SystemKeys.ID_SEPARATOR +
+        SystemKeys.RESOURCE_IDENTITY_PROVIDER +
+        SystemKeys.ID_SEPARATOR +
+        SystemKeys.AUTHORITY_SPID;
+
+    // core
+    private String entityId;
+
+    // <Signature> options
+    private String signingKey;
+    private String signingCertificate; // for <KeyDescriptor use="signing"><KeyInfo>
+
+    // <AssertionConsumerService> options - Currently only one supported at index 0, and binding MUST be post according to SPID specs, and must be tagged with isDefault="true"
+    //    private final Integer ssoServiceIndex = 0;
+    //    private final String ssoServiceBinding = "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST";
+    //    private String ssoServiceLocation;
+
+    // <SingleLogoutService> options
+    //    private String ssoLogout;
+    //    private String ssoLogoutBinding;
+    //    private String ssoLogoutResponseLocation; //optional according to specs
+
+    // <Organization> options
+    private String organizationDisplayName; // TODO: forse questa dovrebbe essere una map[string]string in cui la chiave è la lingua: per ora lang="it" only
+    private String organizationName;
+    private String organizationUrl;
+
+    // <ContactPerson> options
+    // TODO: second ContactPerson tag for private SP currently not supported
+    private String contactPerson_Telephone; // without spaces, includes internation prefix (such as +39 for Italy)
+    private String contactPerson_EmailAddress;
+    private String contactPerson_IPACode;
+    private Boolean contactPerson_Public; // Public/Private
+    private String contactPerson_Type; // "other" (mandatory), optionally includes "billing" (unless private SP, in which case "billing" is also mandatory)
+
+    // AAC options
+    // NOTE: only one among {idps, idpMetadataUrl} can be non null
+    // NOTE: idps is intended to be a subset of the local SPID registry. Both idps and idpMetadataUrl are intended
+    //  to be used for testing purposes. If both are null, the full local SPID registry will be used instead.
+    private Set<String> idps; // optional (see note above)
+    private String idpMetadataUrl; // optional (see note above)
+    private Set<SpidAttribute> spidAttributes; // optional
+    private SpidAuthnContext authnContext; // mandatory
```

Comment:
add validation annotations to mandatory fields (NotNull/NotBlank etc). Also add regex validation (Pattern) to mailAddress,phone,ipacode etc